plugin-server/src/main/ingestion-queues/session-recording/services/replay-events-ingester.ts

```diff
@@ -130,7 +132,18 @@ export class ReplayEventsIngester {
                 // the replay record timestamp has to be valid and be within a reasonable diff from now
                 if (replayRecord !== null) {
                     const asDate = DateTime.fromSQL(replayRecord.first_timestamp)
-                    if (!asDate.isValid || Math.abs(asDate.diffNow('months').months) >= 0.99) {
+                    if (!asDate.isValid || Math.abs(asDate.diffNow('day').days) >= 7) {
```

Comment:
Is there any impact of this change?

Reply:
i don't _think_ there is... if we were really receiving these then and they were playable then folk would be reporting recordings that are many days long (since we sort the timestamps and use them for the player timeline)